target_tools/networking.py.bck

```diff
@@ -11,7 +11,7 @@
 """Networking related functions"""
 import requests
 
-
+# TODO - this might be specific to JS
```

Comment:
Is this file `networking.py.bck` included by mistake?

Reply:
good catch - removed